src/linker/Linker/DirectoryAssemblyResolver.cs

```diff
@@ -71,7 +64,8 @@ protected AssemblyDefinition GetAssembly (string file, ReaderParameters paramete
 
 		public virtual AssemblyDefinition Resolve (AssemblyNameReference name)
 		{
-			return Resolve (name, new ReaderParameters ());
+			defaultReaderParameters.AssemblyResolver = this;
```

Comment:
Why not assign the resolver in the constructor?

Reply:
Right 😄.